internal/deb/extract.go

```diff
@@ -294,3 +373,47 @@ func parentDirs(path string) []string {
 	}
 	return parents
 }
+
+func newTarMetadata() tarMetadata {
+	return tarMetadata{
+		HardLinkRevMap: make(map[string]hardLinkRevMapEntry),
+	}
+}
```

Comment:
You probably don't need this function since we are using it once and it's quite minimal.